Automattic/wp-calypso
# Backup: Add track event when user clicks on confirm restore

The idea is to start tracking when a user clicks on `Confirm restore` when trying to restore a site.

## Proposed Changes
* Add `calypso_jetpack_backup_restore_confirm` event to track when a user clicks on `Confirm restore` in the restore confirmation page.

![CleanShot 2023-09-11 at 19 33 30@2x](https://github.com/Automattic/wp-calypso/assets/1488641/20d9d537-92f9-4ff9-a572-c26a10c43f0a)

## Testing Instructions
* Start a Jetpack Cloud or Calypso Live branch.
* Pick a site that has a Jetpack VaultPress Backup plan and server credentials set.
* Pick one of your backups and click on `Restore to this point`.
* Validate that you see a `t.gif` call with the event `calypso_jetpack_backup_restore_confirm` when you click on `Confirm restore`.

## Pre-merge Checklist
